build.gradle

```diff
@@ -60,6 +62,10 @@ checkstyle {
   maxWarnings 0
 }
 
+compileJava {
+  options.encoding = "UTF-8"
```

Comment:
This still doesn't fix the test runners... No idea how to make those use UTF-8 encoding on my system. :(

Reply:
Resolved by removing UTF-8 characters... 😓